test/utils/setup.ts

```diff
@@ -111,12 +111,8 @@ export const migrationContract = async () => {
     return await hre.ethers.getContractFactory("Migration");
 };
 
-export const migrationContractTo150 = async () => {
-    return await hre.ethers.getContractFactory("Safe150Migration");
-};
-
-export const migrationContractFrom130To141 = async () => {
-    return await hre.ethers.getContractFactory("Safe130To141Migration");
+export const safeMigrationContract = async () => {
+    return await hre.ethers.getContractFactory("SafeMigration");
```

Comment:
Since you have the deployments, I think you should use it here:

```suggestion
    const SafeMigrationDeployment = await deployments.get("SafeMigration");
    const SafeMigration = await hre.ethers.getContractAt("SafeMigration", SafeMigrationDeployment.address);
    return SafeMigration;
```

Then, in the test, you can deploy other copies of `SafeMigration` contract for 1.4.1 for example.

Reply:
I think it is **super** important to test with the actual migration deployment - it would have caught the incorrect configuration in the deployment script.